clients/fides-js/src/extensions/setup.ts

```diff
@@ -0,0 +1,16 @@
+import type { FidesOptions } from "../lib/consent-types";
+
+/**
+ * The path where the GPP extension bundle is located,
+ * likely served by the privacy center
+ */
+const GPP_EXT_PATH = "/gpp-ext.js";
+
+export const setupExtensions = (options: FidesOptions) => {
+  if (options.gppEnabled) {
+    import(GPP_EXT_PATH).catch((e) => {
```

Comment:
this is the dynamic importing step!

Reply:
hm yeah I think it'll try to use example.com/gpp-ext.js. how do we want to treat this? should we try to force it to look relative, or allow the whole path to be configurable?